PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-database/src/main/java/com/microsoft/azure/toolkit/intellij/database/sqlserver/connection/SqlServerDatabaseResourceDefinition.java

```diff
@@ -38,7 +39,13 @@ public MicrosoftSqlDatabase getResource(String dataId) {
     public List<Resource<MicrosoftSqlDatabase>> getResources(Project project) {
```

Comment:
not sure whether we should create a new getResources method which will call getResources and handle exception